symbol.md

```diff
@@ -0,0 +1,181 @@
+# SYMBOLS#PROCS
```

Comment:
@andela-ooranagwa Let me explain in detail. Let's assume the title of the blog post is `The Mystery behind Symbols#to_proc`. This post can have several subsections eg
- Mission Impossible => Introduction
- The Soldier Class
- The Army => soldier collection
- Shorthand Notation: Symbol#to_proc
- It's All about to_proc method: Custom class
- Conclusion.
This is just an example, the subsections you may come up with can be different.